components/studies/ui/src/main/resources/PhenoTips/StudyBindingClass.xml

```diff
@@ -551,7 +551,7 @@
   vertical-align: middle;
 }
 .study-assignment-config {
-  width: 48%;
+  max-width: 48%;
```

Comment:
I think this can be removed completely. Initially permissions had half and studies had half, so that anything coming after them would be forced to be on the next line, but it's simpler to just use `clear:both` on the following item.